### ISSUE TITLE
Errors on latest docs site (7.1.0)

### ISSUE THREAD

**Issue by halvves** [erikras/redux-form#3504]
### Are you submitting a **bug report** or a **feature request**?
Bug

### What is the current behavior?
On the newest version of docs at [https://redux-form.com/7.1.0/examples/material-ui/]( https://redux-form.com/7.1.0/examples/material-ui/) I am getting an endless spinner with error:

```
bundle.js:1 Uncaught Error: Cannot find module "react-dom/lib/EventPluginHub"
    at bundle.js:1
    at e.exports (bundle.js:1)
    at Object.<anonymous> (bundle.js:1)
    at t (bundle.js:1)
    at Object.<anonymous> (bundle.js:1)
    at t (bundle.js:1)
    at bundle.js:1
    at bundle.js:1
```

### What is the expected behavior?
Something like: [https://redux-form.com/7.0.4/examples/material-ui/](https://redux-form.com/7.0.4/examples/material-ui/)

### What's your environment?
MacOS Sierra (10.12.6)
Chrome Version 61.0.3163.100

**Comment by erikras** [comment]
Hmm.. That would be.... _suboptimal_....

Will investigate. 🔍

**Comment by erikras** [comment]
Yeah, this was fixed, but only [in the 7.1.2 docs](https://redux-form.com/7.1.2/examples/material-ui/). Going back to re-publish the fix to 7.1.1 would be more trouble than I think it's worth.
